modules/internal/ChapelBase.chpl

```diff
@@ -1286,23 +1306,29 @@ module ChapelBase {
     __primitive("/=", lhs, rhs);
   }
   inline proc /=(ref lhs:real(?w), rhs:real(w)) {
+    if (chpl_checkDivByZero) then
+      if rhs == 0.0 then
+        halt("Attempt to divide by zero");
```

Comment:
I'm going to add `%(param a: uint(64), b: uint(64))` back in for this purpose, with a TODO comment to remove once param coercion for uints is improved.